ocean_lib/models/arguments.py

```diff
@@ -18,6 +20,28 @@
 logger = logging.getLogger("ocean")
 
 
+class FeeTokenArguments:
+    def __init__(
+        self,
+        address: Optional[str] = None,
+        token: Optional[str] = None,
+        amount: Optional[int] = 0,
+    ):
+        self.address = (
+            Web3.toChecksumAddress(address.lower()) if address else ZERO_ADDRESS
+        )
+        self.token = Web3.toChecksumAddress(token.lower()) if token else ZERO_ADDRESS
+
+        self.amount = amount
+
+    def to_tuple(self):
+        return (
+            Web3.toChecksumAddress(self.address.lower()),
+            Web3.toChecksumAddress(self.token.lower()),
+            self.amount,
+        )
+
+
```

Comment:
I didn't see anywhere in this PR the changes to fee "getters" calls to smart contracts, where the getter call currently returns a 3-tuple. We should make it where the getter call returns the fees object.